src/itest/resources/features/delete-company-profile.feature

```diff
@@ -46,7 +46,21 @@ Feature: Delete company profile
     And the company profile database is down
     When a DELETE request is sent to the company profile endpoint for "<company_number>"
     Then the response code should be 503
+    And the CHS Kafka API is not invoked
 
     Examples:
       | company_number |
-      | 00006400       |
\ No newline at end of file
+      | 00006400       |
+
+
+  Scenario Outline: Delete psc statement when kafka-api is not available
+    Given Company profile api service is running
+    And the company profile resource "<data_file>" exists for "<company_number>"
+    And CHS kafka API service is unavailable
+    When a DELETE request is sent to the company profile endpoint for "<company_number>"
+    Then the response code should be 503
+    And the company profile resource "<data_file>" exists for "<company_number>"
```

Comment:
This step actually saves the contents of that file to the repository, I think the step you want is 'And a company profile exists with id "<companyNumber>"'

Reply:
This still needs to change unless I'm missing something